banzai/main.py

```diff
@@ -198,13 +212,23 @@ def reduce_night():
                                                                  'critical', 'fatal', 'error'])
     parser.add_argument('--post-to-archive', dest='post_to_archive', action='store_true',
                         default=False)
+    parser.add_argument('--post-to-elasticsearch', dest='post_to_elasticsearch', action='store_true',
+                        default=False)
+    parser.add_argument('--elasticsearch-url', dest='elasticsearch_url',
+                        default='http://elasticsearch.lco.gtn:9200')
     parser.add_argument('--fpack', dest='fpack', action='store_true', default=False,
                         help='Fpack the output files?')
 
     parser.add_argument('--rlevel', dest='rlevel', default=91, help='Reduction level')
     parser.add_argument('--db-address', dest='db_address',
                         default='mysql://cmccully:password@localhost/test',
                         help='Database address: Should be in SQLAlchemy form')
+    parser.add_argument('--es-url', dest='elasticsearch_url', default='http://elasticsearch.lco.gtn:9200',
```

Comment:
Looks like a duplicate argument of line 217, --elasticsearch-url

Reply:
Yes. Done.